source/boundary_velocity/ascii_data.cc

```diff
@@ -107,7 +121,11 @@ namespace aspect
           {
             this->scale_factor               /= year_in_seconds;
           }
-
+        prm.enter_subsection("Ascii data model");
+        {
+          use_spherical_unit_vectors = prm.get_bool("Use spherical unit vectors");
+        }
+        prm.leave_subsection();
```

Comment:
In the description of the plugin at the bottom, can you add a line that velocity components can be specified using cartesian  (default) or spherical unit vectors?